src/main/scala/edu/ie3/simona/service/weather/WeatherSourceWrapper.scala

```diff
@@ -401,7 +401,7 @@ private[weather] object WeatherSourceWrapper extends LazyLogging {
       )
   }
   case object WeightSum {
```

Comment:
```suggestion
  object WeightSum {
```
No need for a case object here. Change this as well?